lib/routes/flashcat/blog.ts

```diff
@@ -0,0 +1,61 @@
+import { Route, Data } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+
+export const route: Route = {
+    path: '/blog',
+    categories: ['blog'],
+    example: '/blog',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['flashcat.cloud/blog'],
+            target: '/blog',
+        },
+    ],
+    name: 'Flashcat 快猫星云博客',
```

Comment:
```suggestion
    name: '快猫星云博客',
```
Route name should not include namespace's name.